functions/src/constellation.ts

```diff
@@ -9,85 +8,72 @@ let currentInstance: Constellation;
  * Handles all communication with Constellation throughout the project.
  */
 export class Constellation {
-    libURL: string
-
-    /**
+  /**
    * @constructor
    */
-    constructor() {
-      dag4.network.config({
-        id: config().dag_network.id,
-        beUrl: config().dag_network.be_url,
-        lbUrl: config().dag_network.lb_url,
-      });
-
-      dag4.di.useFetchHttpClient(fetch);
-
-      this.libURL = dag4.network.getNetwork().lbUrl;
-    }
+  constructor() {
+    dag4.di.useFetchHttpClient(fetch);
```

Comment:
This is no longer necessary. You can remove this line.

Reply:
We could remove it from lattice-api too then